caluma-demo/app/components/navbar.js

```diff
@@ -0,0 +1,25 @@
+import { inject as service } from "@ember/service";
+import Component from "@ember/component";
+import config from "../config/environment";
+import { action } from "@ember/object";
+
+export default class NavbarComponent extends Component {
+  @service intl;
+
+  languages = config.languages;
+  fallbackLanguage = config.fallbackLanguage;
+
+  @action
+  setLanguage(language) {
+    if (this.languages.includes(language)) {
+      localStorage.setItem("language", language);
+
+      this.intl.setLocale([
+        localStorage.getItem("language"),
```

Comment:
It is not necessary to access the `localStorage` again since at this point it will always be `language`.

Reply:
I've replaced the `localStorage` with `language`